src/components/Synthetics/MarketCard/MarketCard.tsx

```diff
@@ -47,7 +48,9 @@ export function MarketCard({ marketInfo, allowedSlippage, isLong, isIncrease }:
     if (!marketInfo) return {};
 
     return {
-      liquidity: getAvailableUsdLiquidityForPosition(marketInfo, isLong),
+      liquidity: getAvailableUsdLiquidityForPosition(marketInfo, isLong).gt(0)
+        ? getAvailableUsdLiquidityForPosition(marketInfo, isLong)
```

Comment:
let's put `getAvailableUsdLiquidityForPosition(marketInfo, isLong)` into the variable

Reply:
Updated!